coursedata/adventures/de.yaml

```diff
@@ -1015,12 +1076,14 @@ adventures:
 
                     With `ask` you can make a choice, and with `echo` you can repeat that choice.
 
+                example_code: |
                     ## Example Hedy code
 
+
                     ```
                     print what do you choose?
                     ask choose from rock, paper or scissors
-                    echo so your choice was: 
+                    echo so your choice was:
```

Comment:
Keep that space, so that the echoed choice won't be glued to the colon!
```suggestion
                    echo so your choice was: 
```

Reply:
Will be replaced later on anyway though. Before starting the next adventure, I always copy the remainder of the English file. So, it is fixed there already, I'll copy it over soon anyway.

Reply:
So, it'd be broken from when this translation here (turtle adventure) will be merged until the adventure after that will be translated _and_ merged? If so, is there any reason to simply _not_ remove the space here, so that merging this PR won't break this in the first place? I don't think we should knowingly introduce regressions to the `main` branch and the production website, if we can easily avoid it.

Reply:
On second thought, I think you're totally right. It was rather about not spoiling the commit history (and maybe getting into trouble with merges later). However, in the English language file, it's written without the space. Correcting it now here makes no sense to me, given that I want to copy the original English text over before starting a new translation. And still, I totally dislike introducing an error here. I am uncertain how to proceed.

Reply:
Hi @DerSibi! Simply removing the space will fix the issue as I understand it? If you want to you can commit the fix from the browser:

![image](https://user-images.githubusercontent.com/1003685/153650152-3aa60a99-b468-42b6-8247-00e169603638.png)

I can also do it for you if you want. And don't worry all too much about introducing errors, all code in adventures will be tested before we deploy it.

You can see here that all the tests are ok for this translation:

![image](https://user-images.githubusercontent.com/1003685/153650121-757ee555-a323-4b77-9217-057b0865a644.png)

Reply:
Hi @Felienne, 
thanks for hooking in. It's rather a general question. @das-g and me agree, that there should be a white space in general. However, the EN file does not contain a white space there. And I would like to keep the English parts in the DE-file (that are yet to be translated to German) identical to the EN-file. Even if there are small errors in it. The reason for this is all and foremost, that I follow your advice to copy remaining English parts into the DE file from the EN file before translating a new adventure to have the latest changes.

So, fixing the white space in the DE file now doesn't make sense to me, because I'd overwrite it again before starting the next adventure - or having to fix it manually over and over again. Also, there's a handful of further places in this file below this line with similar issues. So, what's your opinion: Fix the whitespace error or leave it that way?

Reply:
Sorry, I wasn't aware the spaces were now also missing in the English version, and thus misunderstood you, @DerSibi: I thought you were arguing that copying the English file would add the spaces back in later, so we wouldn't have to do so in this PR. Now I understand that you were making us aware that adding them back in within this PR would be somewhat moot, as copying the English file would remove them again later.

We definitely should fix that in the English file. (Or should the behavior of the `echo` command be modified to output a space automatically between its argument and the echoed input, at least in Hedy Levels that don't require string quoting? Requiring trailing spaces in Hedy code for producing the usually expected output might be confusing, as one can't see them in Hedy's code input field.)

Reply:
+1 regarding urge to fix it in EN file. But for this PR, I'd resolve this point.